uqcsbot/base.py

```diff
@@ -131,7 +128,6 @@ def run(self, api_token, verification_token, **kwargs):
         self._api_token = api_token
         self._client = SlackClient(api_token)
         self._verification_token = verification_token
-        self._server = SlackServer(verification_token, '/uqcsbot/events', self, None)
```

Comment:
where is the endpoint defined now?

Reply:
Accidentally deleted - I'd started work on other stuff, rip me